add Documentation tag to systemd unit file;
start rsync after network.target.
Last-Update: 2019-01-16

--- rsync-3.1.3.orig/packaging/systemd/rsync.service
+++ rsync-3.1.3/packaging/systemd/rsync.service
@@ -1,6 +1,8 @@
 [Unit]
 Description=fast remote file copy program daemon
+Documentation=man:rsync(1) man:rsyncd.conf(5)
 ConditionPathExists=/etc/rsyncd.conf
+After=network.target
 
 [Service]
 ExecStart=/usr/bin/rsync --daemon --no-detach
